src/AppInstallerCLICore/Workflows/ShellExecuteInstallerHandler.cpp

```diff
@@ -15,7 +15,7 @@ namespace AppInstaller::CLI::Workflow
         // ShellExecutes the given path.
         std::optional<DWORD> InvokeShellExecute(const std::filesystem::path& filePath, const std::string& args, IProgressCallback& progress)
         {
-            AICLI_LOG(CLI, Info, << "Starting installer. Path: " << filePath);
+            AICLI_LOG(CLI, Info, << "Starting installer: '" << filePath.u8string() << "' with arguments '" << args << "'" );

```

Comment:
>"'" [](start = 118, length = 3)

'\'' ?

Reply:
Hoisted by my own petard.

---
In reply to: [519110975](https://github.com/microsoft/winget-cli/pull/633#discussion_r519110975) [](ancestors = 519110975)